packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -19,6 +19,25 @@ import SidebarNavigationItem from '../sidebar-navigation-item';
 import { PRELOADED_NAVIGATION_MENUS_QUERY } from './constants';
 import { useLink } from '../routes/link';
 
+// Copied from packages/block-library/src/navigation/edit/navigation-menu-selector.js.
```

Comment:
Note: I'm happy with this duplication. However let's track it in https://github.com/WordPress/gutenberg/issues/51455